### PR TITLE
improve docs about jQuery removal

### PR DESCRIPTION
**Pull request by Johann-S**
Fixes part of #28342

### CODE REVIEW

File: site/content/docs/4.3/getting-started/javascript.md [hunk 1]
```diff
@@ -64,6 +64,19 @@ myModal.addEventListener('show.bs.modal', function (e) {
 })
 {{< /highlight >}}
 
+{{< callout warning >}}
+## jQuery events
+
+Bootstrap detect jQuery only if `jQuery` is present in `window` object. If jQuery is found, Bootstrap will emit events thanks to jQuery event system.
```
Comment (yakov116) [inline site/content/docs/4.3/getting-started/javascript.md]:
```suggestion
Bootstrap will detect jQuery only if `jQuery` is present in `window` object. When jQuery is detected, Bootstrap will emit events thanks to jQuery event system.
```

**Review by yakov116 — CHANGES_REQUESTED** [review]

File: site/content/docs/4.3/getting-started/javascript.md (hunk 1 above)
Comment (XhmikosR) [reply site/content/docs/4.3/getting-started/javascript.md]: Thanks. I noticed a few other things in this sentence.

File: site/content/docs/4.3/components/navs.md [hunk 2]
```diff
@@ -586,24 +592,39 @@ Activates a tab element and content container. Tab should have either a `data-ta
 </div>
 
 <script>
-  $(function () {
-    $('#myTab li:last-child a').tab('show')
+  window.addEventListener('load', function () {
```
Comment (XhmikosR) [inline site/content/docs/4.3/components/navs.md]: Is this really needed?

Comment (Johann-S) [reply site/content/docs/4.3/components/navs.md]: No it was just to match the previous example

Comment (XhmikosR) [reply site/content/docs/4.3/components/navs.md]: Maybe we should remove it then?

Comment (Johann-S) [reply site/content/docs/4.3/components/navs.md]: sure 👍

**Review by XhmikosR — APPROVED** [review]

**Pull request merged**